Test/git-issues/git-issue-859a.dfy.expect

```diff
@@ -1,7 +1,7 @@
-git-issue-859a.dfy(28,2): Error: The Extract member may not be ghost unless the initial LHS is ghost
-git-issue-859a.dfy(28,4): Error: The IsFailure member may not be ghost (type FailureCompatible used in :- statement)
-git-issue-859a.dfy(28,4): Error: The PropagateFailure member may not be ghost (type FailureCompatible used in :- statement)
-git-issue-859a.dfy(33,2): Error: The Extract member may not be ghost unless the initial LHS is ghost
-git-issue-859a.dfy(44,9): Error: The IsFailure member may not be ghost (type FailureCompatible used in :- statement)
-git-issue-859a.dfy(44,9): Error: The PropagateFailure member may not be ghost (type FailureCompatible used in :- statement)
+git-issue-859a.dfy(34,4): Error: The Extract member may not be ghost unless the initial LHS is ghost
+git-issue-859a.dfy(44,7): Error: a call to a ghost function is allowed only in specification contexts (consider declaring the function with 'function method')
+git-issue-859a.dfy(55,4): Error: return statement is not allowed in this context (because it is guarded by a specification-only expression)
+git-issue-859a.dfy(82,10): Error: Member 'IsFailure' in type 'NotFunctionsAtAll' (used indirectly via a :- statement) is expected to be a function
+git-issue-859a.dfy(82,10): Error: Member 'PropagateFailure' in type 'NotFunctionsAtAll' (used indirectly via a :- statement) is expected to be a function
+git-issue-859a.dfy(82,10): Error: Member 'Extract' in type 'NotFunctionsAtAll' (used indirectly via a :- statement) is expected to be a function
```

Comment:
The error messages are so different -- are we actually testing the same functionality or error conditions?

Reply:
I had rewritten this test file substantially, and hence the expected output is also quite different.

Reply:
OK